File: backend/AnimePlay/src/main/java/com/backend/animeplay/entity/Anime.java
package com.backend.animeplay.entity;

import jakarta.persistence.*;
import lombok.*;
import lombok.experimental.FieldDefaults;

@Entity
@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class Anime {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Integer id;

    String title;
    @Column(columnDefinition = "TEXT")
    String description;
    String year;
    String genre;

    String posterUrl;
    String trailerUrl;
}

File: donet/AnimePlay/Entities/Anime.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AnimePlay.Entities
{
    [Table("Animes")]
    public class Anime
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public string? Title { get; set; }

        [Column(TypeName = "text")] 
        public string? Description { get; set; }

        public string? Year { get; set; }

        public string? Genre { get; set; }

        public string? PosterUrl { get; set; }

        public string? TrailerUrl { get; set; }
    }
}